src/Everscale/Address.h

```diff
@@ -0,0 +1,47 @@
+// Copyright © 2017-2021 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../Data.h"
+#include "../PublicKey.h"
+
+#include <optional>
+#include <string>
+
+namespace TW::Everscale {
+
+class Address {
+public:
+    /// Number of bytes in an address
+    static const size_t size = Hash::sha256Size;
+
+    using MsgAddressInt = std::pair<int8_t, std::array<byte, Address::size>>;
+
+    std::int8_t wc_;
+    std::array<byte, size> address_{};
+
+    /// Determines whether a string makes a valid address.
+    static bool isValid(const std::string& string);
```

Comment:
should be `[[nodiscard]]` and `noexcept` after modifying `parseWorkChainId`

Reply:
Done